### DISCUSSION TITLE
BT=SV Alarm

### DISCUSSION THREAD

**Discussion by fwrowe** [artisan-roaster-scope/artisan#1414] [Q&A]
When I am heating up my roaster, I set SV to 10 degrees Celsius higher than the target temperature and use automatic heating. When BT rises to SV, I start a 20-minute countdown and wait for the countdown to end. After that, I will lower SV by 20 degrees Celsius lower than the target temperature, and when BT drops to SV, I will raise SV to the target temperature again. When BT rises to SV, I will put the coffee beans into the roaster. Currently, I can only stare at the screen to see if BT is equal to SV for this operation, so I hope there is a function to remind me when BT reaches SV with a sound prompt, so I don’t have to stare at the screen all the time.

**Comment by MAKOMO** [comment]
Did you have a look at [Alarms](https://artisan-roasterscope.blogspot.com/2013/03/alarms.html)?

What you describe sounds like a between patch protocol. Did you check our post on [Supporting Between Batch Protocols](https://artisan-roasterscope.blogspot.com/2020/12/supporting-between-batch-protocols.html)
